test/runtests.jl

```diff
@@ -12,7 +12,18 @@ else
 end
 
 if VERSION >= v"0.5.0"
-    @test isempty(detect_ambiguities(ColorTypes, Base, Core))
+    if isdefined(Core, :UnionAll)
```

Comment:
not the right condition for the introduction of the allow_bottom kwarg which was introduced much later